apis/python/src/tiledbsoma/_read_iters.py

```diff
@@ -148,7 +152,7 @@ def __init__(
     @classmethod
     def _validate_args(
         cls,
-        shape: NTuple,
+        shape: Union[NTuple, list[int]],
```

Comment:
Is there a reason that this needs to be specifically a `list`? It seems like it could be a `Sequence` like the rest of the analogous arguments.